src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -176,4 +185,17 @@ private static async Task AwaitCancelKeyPress()
         await tcs.Task;
     }
 
+    private static S? Select<T, S>(this T? value, Func<T, S> mapper) where T : struct where S : struct
+    {
+        return value != null ? new S?(mapper(value.Value)) : null;
+    }
+
+    private static IEnumerable<S> Selecti<T, S>(this IEnumerable<T> values, Func<int, T, S> mapper)
```

Comment:
There's an existing version of Select in Linq that does this: https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.select?view=netframework-4.8#System_Linq_Enumerable_Select__2_System_Collections_Generic_IEnumerable___0__System_Func___0_System_Int32___1__

Reply:
Fixed, thanks